src/lucky_migrator/create_table_statement.cr

```diff
@@ -16,19 +19,32 @@ class LuckyMigrator::CreateTableStatement
     with self yield
     process_rows
     statement << ")"
+    process_indices
     statement.to_s
   end
 
   private def process_rows
     statement << rows.join(",\n")
   end
 
-  macro add(type_declaration)
+  private def process_indices
+    return if indices.empty?
+    statement << "\n"
+    statement << indices.join("\n")
+  end
+
+  # Generates raw sql from a type declaration and options passed in as named
+  # variables.
+  macro add(type_declaration, index = false, using = "btree", unique = false)
     {% if type_declaration.type.is_a?(Union) %}
       add_column :{{ type_declaration.var }}, {{ type_declaration.type.types.first }}, optional: true
     {% else %}
       add_column :{{ type_declaration.var }}, {{ type_declaration.type }}
     {% end %}
+
+    {% if index %}
```

Comment:
I'd love to make it so that you only need to pass `unique: true` since it would't make sense to do `index: false, unique: true`. So I think this would be something like `if index || unique`

Reply:
Ah nice I didn't notice that, I made the change.